web/source/kmwcallback.js

```diff
@@ -54,7 +54,7 @@
         Lelem._KeymanWebSelectionEnd=Le;
         keymanweb._IgnoreNextSelChange = 0;
         if(Ptext!=null)keymanweb.KO(0, Lelem, Ptext);
-        if(typeof(PdeadKey)!='undefined') keymanweb.KDO(0, Lelem, PdeadKey);
+        if((typeof(PdeadKey)!=='undefined') && (PdeadKey !== null)) keymanweb.KDO(0, Lelem, PdeadKey);
```

Comment:
I assume this is something that Closures complained about? Because that's potentially a change in functionality.